src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/NomisApiService.kt

```diff
@@ -80,5 +81,21 @@ class NomisApiService(
 
   private fun String.nomisRoleName(): String = take(30)
 
+  fun getUserRoles(username: String): UserRoleDetail {
+    log.debug("request dps roles for {}", username,)
```

Comment:
this logging might be redundant since the Request and parameter will get logged anyway